client/rpcserver/handlers.go

```diff
@@ -768,4 +801,49 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + logoutStr + `"`,
 	},
+	orderBookRoute: {
+		argsShort:  `"host" base quote nOrders`,
+		cmdSummary: `Retrieve all orders for a market.`,
+		argsLong: `Args:
+    host (string): The DEX to retrieve the order book from.
+    base (int): The BIP-44 coin index for the market's base asset.
+    quote (int): The BIP-44 coin index for the market's quote asset.
+    nOrders (int): Optional. Default is 0. The number of orders from the top of
+      buys and sells to return. 0 returns all orders. Epoch orders are not
+      truncated.`,
+		returns: `Returns:
+    obj: A map of orders.
+    {
+      "sells" (array): An array of booked sell orders.
+      [
+        {
+          "qty" (float): The number of coins base asset being sold.
+          "rate" (float): The coins quote asset to pay per coin base asset.
+          "sell" (bool): Always true because this is a sell order.
+          "token" (string): The first 8 bytes of the order id, coded in hex.
+        },...
+      ],
+      "buys" (array): An array of booked buy orders.
+      [
+        {
+          "qty" (float): The number of coins base asset being bought.
+          "rate" (float): The coins quote asset to accept per coin base asset.
+          "sell" (bool): Always false because this is a buy order.
+          "token" (string): The first 8 bytes of the order id, coded in hex.
+        },...
+      ],
+      "epoch" (array): An array of epoch orders. Epoch orders include new limit
+        orders that have not been booked yet. They also contain orders that are
+	never booked: cancellations, market orders, and limit orders with
+	immediate=true. They are not truncated.
```

Comment:
Even limit orders with time in force "standing" (immediate=false) may not be booked if they are completely filled on match, so "yet" is not accurate.  I'd just say that this array includes all kinds of orders, even those that cannot or may not be booked.